Krzywicki_reflection_1.qmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "Self-reflection-1-krzywicki"
+format: html
+---
+
+<span style="color: blue;">Questions About you
+
+Please tell me a little bit about your research interests and how you imagine integrating spatial data in your research.
+
+I am studying how global land use change affects local sense of place through the lens of migratory networks. I envision using spatial data to compare and contrast sites along migratory bird networks.
+
+How do you learn best, and what do you feel is your biggest strength as a student?
+
+I am more of a words person than a visual learner, so maps don't always come the easiest to me! I am excited to push myself this semester.
```

Comment:
I will keep this in mind! Let me know if verbal interpretations/walkthroughs of maps would help you at any point in the semester!